### PR TITLE
Add pattern utils for FIO

### PR DESCRIPTION
**Pull request by Ostrokrzew**

### CODE REVIEW

File: test_tools/fio/fio_patterns.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    for i in range(0, 256):
+        seq_pattern += f"{hex(i).lstrip('0x'):0>2}"
+    return seq_pattern.strip()
+
+
+def _get_random_pattern():
+    random_pattern = "0x"
+    for i in random.sample(range(256), random.randint(1, 256)):
+        random_pattern += f"{hex(i).lstrip('0x'):0>2}"
+    return random_pattern.strip()
```
Comment (katlapinka) [inline test_tools/fio/fio_patterns.py:21]: You can move those methods to Pattern class

File: test_tools/fio/fio_patterns.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    bin_2 = "0x55"
+    random = _set_random_pattern()
+
+    def get_unique_patterns():
+        return set(list(Pattern.__members__.values()))
+
+    def mix_patterns():
+        patterns = list(Pattern.get_unique_patterns())
+        random.shuffle(patterns)
+        return patterns
```
Comment (robertbaldyga) [inline test_tools/fio/fio_patterns.py:39]: This doesn't look like generally useful functionality. I think it should be moved to the test which uses it.

File: test_tools/fio/fio_patterns.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    def __str__(self):
+        return self.value
```
Comment (robertbaldyga) [inline test_tools/fio/fio_patterns.py:42]: Do we really need to override default method? This is quite a standard to obtain enum value by `Class.item.value` property.

File: test_tools/fio/fio_patterns.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+import random
+from aenum import Enum
+
+
+class Pattern(Enum):
+    def _set_seq_pattern():
+        seq_pattern = "0x"
+        for i in range(0, 256):
+            seq_pattern += f"{hex(i).lstrip('0x'):0>2}"
+        return seq_pattern.strip()
```
Comment (robertbaldyga) [inline test_tools/fio/fio_patterns.py:15]:
```python
return "0x"+"".join([f"{i:02x}" for i in range(0, 256)])
```

File: test_tools/fio/fio_patterns.py [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+    def _set_random_pattern():
+        random_pattern = "0x"
+        for i in random.sample(range(256), random.randint(1, 256)):
+            random_pattern += f"{hex(i).lstrip('0x'):0>2}"
+        return random_pattern.strip()
```
Comment (robertbaldyga) [inline test_tools/fio/fio_patterns.py:21]:
```python
return "0x"+secrets.token_hex(random.randint(1, 256))
```
BTW do we really want this pattern to be variable length?

Comment (Ostrokrzew) [reply test_tools/fio/fio_patterns.py:21]: nope, removed

File: test_tools/fio/fio_patterns.py (hunk 4 above)
Comment (Ostrokrzew) [reply test_tools/fio/fio_patterns.py:15]: thx

**Review by robertbaldyga — APPROVED** [review]

**Pull request merged**